visualizations/vis_class.py

```diff
@@ -0,0 +1,24 @@
+from collections.abc import MutableMapping
+from abc import abstractmethod
+
+
+class Visualization(MutableMapping):
+    @abstractmethod
+    def __init__(self, filename):
+        pass
+
+    @abstractmethod
+    def _import_file(self):
+        """
+        This method takes a JSON file and returns a dictionary of data for the \
+            visualization.
```

Comment:
I don't think you need the `\` character for strings within the `"""` spots. You can erase the ones here and below.